docs/moduleCom1DFA.rst

```diff
@@ -52,6 +52,7 @@ projection:
 
 * release area scenario as (multi-) polygon shapefile (in Inputs/REL; multiple features are possible)
 
+  - the release area polygon should not contain any "holes" or inner rings
```

Comment:
```suggestion
  - the release area polygon must not contain any "holes" or inner rings
```
This is good, but please move it to a separate point and write : Any shapefile with polygons must not....